contrib/kafka/config/SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security
+
+kafka channel controller and dispatcher connect to brokers without
+encryption and authentication by default. Encryption using SSL can be
+turned.
+
+## Connect over SSL(TLS)
+
+1. Create secret from Cluster CA public key to verify the identity of the Kafka brokers.
+
+   ```yaml
+   kubectl create secret generic kafka-certificates  --from-file=ca.crt -n knative-eventing
+   ```
+
+1. Define environmental variables KAFKA_TLS_ENABLE and KAFKA_CA_CERT in their Deployments.
```

Comment:
Can you define a followup to use a ConfigMap and have the Kafka controller/dispatcher read from this ConfigMap rather than needing to update both YAMLs to get security?

I'm concerned about getting one but not the other set correctly, but not concerned enough to block this PR, which is making forward progress.

Reply:
> Can you define a followup to use a ConfigMap and have the Kafka controller/dispatcher read from this ConfigMap rather than needing to update both YAMLs to get security?

Using ConfigMap is good idea and we can use it.
But, I am thinking that 2-way authentication by using client cert and private key could be implemented in the future PR.
So, I am wondering that all certs should be stored in secret (`kafka-certificates`) altogether. (The private key must be in secret.)

Reply:
I change my mind. The CA (in this PR) would be better to specified in ConfigMap rather than secret. Would you like me to update here or send another PR after this PR was accepted?